src/lifecycle/created.js

```diff
@@ -55,7 +55,7 @@ export default function (opts) {
 
   return function () {
     let info = data(this, opts.id);
-    let isNative = this.createdCallback;
+    let isNative = !this.hasOwnProperty('_isPolyfill') && this.createdCallback || !(this._isPolyfill = true);
```

Comment:
Would it be better to store the result of this on the definition (opts): https://github.com/skatejs/skatejs/blob/master/src/index.js#L69?